cluster/gce/gci/configure-helper.sh

```diff
@@ -1352,6 +1352,7 @@ function start-kube-apiserver {
   local audit_policy_config_volume=""
   local audit_webhook_config_mount=""
   local audit_webhook_config_volume=""
+  # Basic audit is deprecated and soon will be removed.
```

Comment:
Print a warning?

Reply:
I think a warning here would be printed on the node and not be user visible. We should add a warning to config-default, config-test, or util.sh which are run client side.

Reply:
Thanks, ack